sycl/include/CL/__spirv/spirv_ops.hpp

```diff
@@ -13,6 +13,13 @@
 #include <cstdint>
 #include <type_traits>
 
+// Convergent attribute
+#ifdef __SYCL_DEVICE_ONLY__
+#define __SYCL_CONVERGENT__ __attribute__((convergent))
```

Comment:
I wonder, why `SYCL`? AFAIK `convergent` is a non-sycl-specific clang attribute.

Reply:
Clang attribute is not SYCL specific, but macro is.
Do you have better name in mind?

Reply:
Not sure. I'd probably just named it as `CONVERGENT`. I also don't see why this macro is SYCL specific. We, for example, have `ALWAYS_INLINE` macro which somehow is not SYCL specific. So, I'll leave final naming up to you.